framework/auth/views.py

```diff
@@ -800,7 +801,6 @@ def unconfirmed_email_add(auth=None):
         'removed_email': json_body['address']
     }, 200
 
-
 def send_confirm_email(user, email, renew=False, external_id_provider=None, external_id=None, destination=None):
```

Comment:
This method is used in 8 different places (excluding import / tests). Please take a look and add QA notes to make sure they cover all affected code during testing. Please note 1 of 8 tests is Institution SSO related and thus not testable by QA, but we have unit test covering it so it should be fine.

![Screenshot 2024-07-12 at 9 31 27 AM](https://github.com/user-attachments/assets/4e368966-8e2b-431b-892c-e15c516e526c)